bikeshed/boilerplate/whatwg/computed-metadata-LS-COMMIT.include

```diff
@@ -0,0 +1,5 @@
+{
+  "Title": "[SPECTITLE] Standard Commit Snapshot [COMMIT-SHA]",
+  "Logo": "https://resources.whatwg.org/logo-[SHORTNAME]-snapshot.svg",
+  "!Commits": "<a href=/ id=commit-snapshot-link>Go to the living standard</a>"
```

Comment:
Is it intentional to remove the other two links? I think it'd be reasonable to keep including them, as we do now.

Reply:
Yeah, it was intentional to include less information that might go stale in snapshots.

Reply:
I mean, the information is a link to GitHub and Twitter... I'm not sure why those would go stale.

Reply:
I guess I don't really see a use case for them, that's why I also didn't include Tests/Participate.

Reply:
Oh, those get deleted if you don't explicitly include them? I thought the defaults would apply.

I'm not super-excited about making the commit snapshots less usable in this way... they're meant to be nice stable references, and letting people navigate from them to the list of latest commits seems quite important, if they want to know e.g. what has changed since the snapshot they're looking at.

Reply:
Oh, I haven't actually tested how that works. It might well work that way.